doc/sphinx/proof-engine/tactics.rst

```diff
@@ -3419,6 +3419,124 @@ the conversion in hypotheses :n:`{+ @ident}`.
 
    This is the most general syntax that combines the different variants.
 
+.. tacn:: with_strategy @strategy_level [ {+ @qualid } ] @tactic
```

Comment:
If you could replace `@tactic` with the correct `@ltac_expr`, it would be more precise.

Reply:
```suggestion
.. tacn:: with_strategy @strategy_level [ {+ @qualid } ] @ltac_expr
```